src/org/labkey/test/tests/AdminConsoleTest.java

```diff
@@ -50,6 +55,75 @@ public String getProjectName()
         return null;
     }
 
+    @Test
+    public void testServerHttpHeaderSetting()
+    {
+        goToAdminConsole().clickSiteSettings();
+        waitForElement(Locator.name("includeServerHttpHeader"));
+        WebElement checkbox = Locator.checkboxByName("includeServerHttpHeader").findElement(getDriver());
+
+        boolean originalValue = checkbox.isSelected();
+
+        // Try with the setting on
+        if (!originalValue)
+            click(Locator.checkboxByName("includeServerHttpHeader"));
+        clickButton("Save");
```

Comment:
This checkbox should be manipulated via `CustomizeSitePage`.
```suggestion
        CustomizeSitePage customizeSitePage = goToAdminConsole().clickSiteSettings();
        boolean originalValue = customizeSitePage.isServerHttpHeaderEnabled();

        // Try with the setting on
        if (!originalValue)
            customizeSitePage.enableServerHttpHeader(true);
        customizeSitePage.save();
```

Reply:
OK. Made another update, including another improvement to the section I used as a reference.